code/modules/admin/verbs/adminhelp.dm

```diff
@@ -251,6 +251,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	ticket_interactions = list()
 	player_interactions = list()
 
+	addtimer(CALLBACK(src, .proc/add_to_ping_ss, 3 MINUTES)) //SKYRAT EDIT ADDITION - Ticket Ping
```

Comment:
This time duration should be done via config too, so it's easy for keyholders to change that duration without having to make it a PR.

Reply:
yep yep, it ready